packages/core/src/components/DataTable/DataTable.tsx

```diff
@@ -465,43 +461,36 @@ export class DataTable extends React.Component<DataTableProps & WithStylesProps,
       sortDirection,
     );
 
+    const tableHeight = autoHeight
+      ? height! - (this.shouldRenderTableHeader() ? getHeight(rowHeight, tableHeaderHeight) : 0)
```

Comment:
`defaultProps.height` is a number, but if it somehow was `undefined` this would be `NaN` 😱 both sides of the ternary actually include a `height!` (the other is buried in the function).

maybe I'll do a sweep to update all `height!` instances to `height || 0` to be a bit safer.

Reply:
ooh that was my fear! better safe than sorry. thanks for updating!